### ISSUE TITLE
[MU4 Issue] Shared notehead playback sounds uneven compared to MS3

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#15643]
**Describe the bug**

Listen to the short guitar excerpt in the attached MS3 file. Notice the shared noteheads and the fact that the volume is consistent throughout. 
[shared_notehead_playback.zip](https://github.com/musescore/MuseScore/files/10320578/shared_notehead_playback.zip)

Now, Play back the same file in MS4.

RESULT: The volume is inconsistent. It sounds as if it is louder on the shared noteheads then diminishes on the single notes following them. 

**Expected behavior**

The volume should be smooth and consistent, as it was in MS3.

**Platform information**

OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-223630304, revision: github-musescore-musescore-3f9b94d

**Comment by rgreen5** [comment]
It sounds as if playback is accentuating the shared noteheads, i.e. playing them at a greater note velocity than single noteheads. This is an error (and a regression from MS3) because the shared notehead does _not_ consist of two separate notes but only a _single_ note.

You can clearly hear the problem in the attached MS4 file below.
[shared_notehead_playback.zip](https://github.com/musescore/MuseScore/files/11082405/shared_notehead_playback.zip)

**Comment by MarcSabatella** [comment]
This isn't necessarily incorrect; it depends on the reason for having written the unison in the first place.  If it's because two otherwise independent voices happen to coincidence on the same pitch but you're only supposed to play it once, then indeed, it shouldn't sound noticeably different.  But, if it's indicating that you should actually play a double stop on a stringed instrument - the same pitch on two different strings at one - then it certainly should sound louder.

Probably there could be a better way of differentiating these cases, but for now, unchecking Play on one of them is as good a method as any.  I suppose it's possible that the act of explicitly sharing the notehead could be taken as a sign that you don't want the double stop.  Not sure how valid that assumption is in practice though.

**Comment by rgreen5** [comment]
A shared notehead usually occurs when a note is to be played with the thumb then allowed to sustain while the notes of the other voice (usually voice 1) are played. So volume should be consistent throughout with no emphasis on the shared notehead (unless of course an articulation indicates otherwose).

Even in the case of (less common) double stops, emphasis is unlikely to be the desired option. So consistent volume is a safe bet here as well.

Unchecking "Play" on one voice isn't really viable because of the amount of extra work required to silence all those notes.

In short, shared noteheads should play at the same volume as other notes. I would suggest giving this issue some priority as guitar scores frequently use shared noteheads.

**Comment by rgreen5** [comment]
SUGGESTION: Revert playback of shared noteheads and/or unisons to the MS3 default for now. This would, at least, fix the problem with guitar scores. Then open a feature request, if needed, for the handling of double stop unisons.
